### PR TITLE
CB-584 Create minimal test run that is part of deployment pipeline

### PR DESCRIPTION
**Pull request by aszegedi**
@lnardai and @attilapalfi92 please review these changes for execution YARN smoke tests on Jenkins. Along with this I've introduced the quick fix for CB-720 and the related updates based on CB-701.

You can check the related Jenkins job at `e2e-test/job/yarn-smoke-test/` path on the team's Jenkins.

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/cloud/v2/gcp/GcpCloudProvider.java [hunk 1]
```diff
@@ -105,7 +113,7 @@ public StackAuthenticationEntity stackAuthentication(StackAuthenticationEntity s
     }
 
     @Override
-    public String getDefaultClusterDefinitionName() {
-        return GcpParameters.DEFAULT_CLUSTER_DEFINTION_NAME;
+    public String getClusterDefinitionName() {
+        return getTestParameter().getWithDefault(CommonCloudParameters.CLUSTER_DEFINITION_NAME, CommonCloudParameters.DEFAULT_CLUSTER_DEFINITION_NAME);
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/cloud/v2/gcp/GcpCloudProvider.java]: Gcp has it's own 'DEFAULT_CLUSTER_DEFINTION_NAME', please use it. Applies to the other cloud providers too.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/cloud/v2/yarn/YarnCloudProvider.java [hunk 2]
```diff
@@ -8,20 +8,26 @@
...
 import com.sequenceiq.it.cloudbreak.newway.entity.VolumeV4Entity;
 import com.sequenceiq.it.cloudbreak.newway.entity.credential.CredentialTestDto;
+import com.sequenceiq.it.cloudbreak.newway.entity.imagecatalog.ImageCatalogTestDto;
 
 @Component
 public class YarnCloudProvider extends AbstractCloudProvider {
 
     public static final String DEFAULT_LOCATION = "Frankfurt";
 
+    public static final String DEFAULT_REGION = "default";
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/cloud/v2/yarn/YarnCloudProvider.java]: Please move all these constants to the YarnParameters class.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/smoke/YarnSmokeTest.java [hunk 3]
```diff
@@ -64,21 +75,53 @@ public void testWhenCreatedYARNClusterShouldBeUpScaled(TestContext testContext)
...
-                .when(StackTestAction::stop)
-                .await(STACK_STOPPED)
-                .when(StackTestAction::start)
+    @Test(dataProvider = TEST_CONTEXT, enabled = false, description = "Stop is not supported on YARN cloudplatform")
+    @Description(
+            given = "a valid YARN cluster",
+            when = "stopping then starting the cluster",
+            then = "the cluster should be available and successfully started"
+    )
+    public void testWhenCreatedYARNClusterStoppedStarted(TestContext testContext) {
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/smoke/YarnSmokeTest.java]: I know it's not up to you, but I really hope that Yarn will be reliable enough to introduce these tests to the build pipeline...

https://media3.giphy.com/media/uq6ILNBI6g3As/giphy.gif

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/smoke/YarnSmokeTest.java [hunk 4]
```diff
@@ -64,21 +75,53 @@ public void testWhenCreatedYARNClusterShouldBeUpScaled(TestContext testContext)
...
+        String clusterName = randomNameCreator.getRandomNameForResource();
+        testContext.given(clusterName, StackTestDto.class)
+                .withName(clusterName)
+                .when(StackTestAction::create, key(clusterName))
```
Comment (attilapalfi92) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/smoke/YarnSmokeTest.java]: Does it work without constantly passing the key? It shouldn't be necessary.

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/smoke/YarnSmokeTest.java (hunk 3 above)
Comment (aszegedi) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/smoke/YarnSmokeTest.java]: @lnardai or @sodre90 may can response this...may can convince you ;)

**Pull request merged**